libs/platform/src/lib/components/value-help-dialog/components/define-tab/define-tab.component.ts

```diff
@@ -38,6 +40,11 @@ class ExtendedExcludedEntity extends VhdExcludedEntity {
   changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class DefineTabComponent<T> extends VhdBaseTab implements OnChanges {
+  protected defaultId = `fd-title-id-${randomId++}`;
+
+  @Input()
+  titleId: string = this.defaultId;
```

Comment:
why not 
```suggestion
  titleId: string = `fd-title-id-${randomId++}`;
```
?

Reply:
why did you revert this? only `string` type needed to be removed to fix the lint, I think.

Reply:
still not addressed

Reply:
did you try removing `string`? The other error at line 43 is a different one, you can fix that too.
p.s: please don't reply by editing my original comments, i won't get notified that you have replied and will miss to see your response. Thanks.